From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Rong Bao <rong.bao@csmantle.top>
Date: Fri, 17 Jul 2026 15:17:35 +0800
Subject: [PATCH 47/50] CSMANTLE: Revert "FROMLIST: rust: generate a fatal
 error if BINDGEN_TARGET is undefined"

This reverts commit 01184c45e4269b7d1b32ead7d450767d1858a379.
---
 scripts/Makefile.rust | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/scripts/Makefile.rust b/scripts/Makefile.rust
index c3b8c2435bae9a02b1e1222a885ade88c2529fb2..fd48b0e0d46fbbb0a79d3fad45586c3f34fb7379 100644
--- a/scripts/Makefile.rust
+++ b/scripts/Makefile.rust
@@ -8,8 +8,4 @@ BINDGEN_TARGET_i386   := i386-linux-gnu
 BINDGEN_TARGET_um     := $(BINDGEN_TARGET_$(SUBARCH))
 BINDGEN_TARGET                := $(BINDGEN_TARGET_$(SRCARCH))
 
-ifeq ($(BINDGEN_TARGET),)
-$(error add '--target=' option to scripts/Makefile.rust)
-else
 export BINDGEN_TARGET
-endif
-- 
2.55.0

